src/algorithms/acquisition/adapters/beidou_b1c_pcps_acquisition.h

```diff
@@ -0,0 +1,197 @@
+/*!
+* \file beidou_B1Cd_pcps_acquisition.h
+* \brief Adapts a PCPS acquisition block to an AcquisitionInterface for
+*  BEIDOU B1C signals
+* \authors <ul>
+*          <li> Andrew Kamble, 2019. andrewkamble88@gmail.com

```

Comment:
Same as before, improve the doc by adding your contact info and a headline about GSoC 2022